src/spacegame3/SpaceGame.java

```diff
@@ -42,22 +45,33 @@
 
     private boolean gameStarted;
 
-    public SpaceGame() {
+    /*
+     *SpaceGame constructor
+     *links startScreen and planetScreen to this context
+     */
+    public SpaceGame()
+    {
         startScreen = new StartScreen(this);
         planetScreen = new PlanetScreen(this);
         gameScheme = null;
         gameStarted = false;
     }
 
+    //game scheme getter
     public GameScheme getGameScheme() {
         return gameScheme;
     }
 
+    //game scheme setter
```

Comment:
use /** */ for methods (just repeating for each one), also don't need to specify that it is a setter, the method name gives this information, but it is interesting to tell about side effects, checks, etc.,
For example, in this case, it might be interesting to say that the ImageLibrary has its context set to the gameScheme StoryTelling path